be/src/storage/lake/lake_primary_index.cpp

```diff
@@ -103,13 +102,13 @@ Status LakePrimaryIndex::_do_lake_load(TabletManager* tablet_mgr, const TabletMe
             RETURN_IF_ERROR(StorageEngine::instance()
                                     ->get_persistent_index_store(metadata->id())
                                     ->create_dir_if_path_not_exists(path));
-            _persistent_index = std::make_unique<LakeLocalPersistentIndex>(path);
+            _persistent_index = std::make_shared<LakeLocalPersistentIndex>(path);
```

Comment:
is there a necessity to perform the load under lock? since there is already a lock defined and used for pk index.

Reply:
It will call when first time initial, no concurrency issue at this phase.